app/views/gas_asset_lists/_table.html.haml

```diff
@@ -1,3 +1,15 @@
+%h2 Gas infrastructure
+.help-block
+  %p
+  In the table below you can edit the specifics of the gas infrastructure of your LES. Three types of network components can be chosen:
```

Comment:
Needs another two spaces in front of the text `In the table...`, so that the content belongs to the `%p` above.

```haml
.help-block
  %p
    In the table below you can edit the specifics of the gas in [...]
```